cli/ts/commands/poll.ts

```diff
@@ -0,0 +1,60 @@
+import { ZeroAddress } from "ethers";
+import { MACI__factory as MACIFactory, Poll__factory as PollFactory } from "maci-contracts";
+
+import type { IGetPollArgs, IGetPollData } from "../utils/interfaces";
+
+import { banner } from "../utils/banner";
+import { logError, logGreen, success } from "../utils/theme";
+
+/**
+ * Get deployed poll from MACI contract
+ * @param {IGetPollArgs} args - The arguments for the get poll command
+ * @returns {IGetPollData} poll data
+ */
+export const getPoll = async ({ maciAddress, signer, pollId, quiet = true }: IGetPollArgs): Promise<IGetPollData> => {
+  banner(quiet);
+
+  const maciContract = MACIFactory.connect(maciAddress, signer);
+  const id =
+    pollId === undefined ? await maciContract.nextPollId().then((nextPollId) => nextPollId - 1n) : BigInt(pollId);
+
+  if (id < 0n) {
+    logError("Invalid poll id");
+  }
+
+  const pollAddress = await maciContract.polls(id);
+
+  if (pollAddress === ZeroAddress) {
+    logError("MACI contract doesn't have any deployed poll");
+  }
+
+  const pollContract = PollFactory.connect(pollAddress, signer);
+
+  const [[deployTime, duration], isStateAqMerged, numSignups] = await Promise.all([
+    pollContract.getDeployTimeAndDuration(),
+    pollContract.stateAqMerged(),
+    pollContract.numSignups(),
```

Comment:
so the number of signups will only be known after the state aq is merged. what about taking it from maci instead which will have up to date info if the poll is not completed yet?

Reply:
or maybe to be even more precise, we first can check if the state aq are merged, if they are, take from poll. if not take from maci

Reply:
Does it update when new poll is deployed? If no, I can just remove it.

Reply:
on maci, numSignups can keep growing poll after poll as new users signup. whereas on a specific poll, we store the number of users signed up up until the poll is finalized